examples/ipv6-dualstack/main.tf

```diff
@@ -47,3 +47,36 @@ module "vpc" {
 
   tags = local.tags
 }
+
+module "vpc_endpoints" {
```

Comment:
we don't need a new example - we can use an existing example

Reply:
hi @bryantbiggs , I have a question, 
as an example - ipv6 VPC endpoints can be created without subnets attached. But as a good example - probably better to add a fully working example.  If the second option is preferable - we need to add ipv6 subnets to the VPC setup. Something like that: 
```
    athena = {
      service                 = "athena"
      ip_address_type = "dualstack"
      subnet_ids           = [ipv6_subnet1, ipv6_subnet2. ipv6_subnet3]
      tags                       = { Name = "athena-vpc-endpoint" }
    },
```
What do you think about this ?

Reply:
apologies, I don't understand the question - what are you asking?

Reply:
sorry, let me rephrase with details

I can add this minimal example, which will create the endpoint but it will not work because subnets are not defined
```
lakeformation = {
  service         = "lakeformation"
  ip_address_type = "ipv6"
  tags            = { Name = "lakeformation-vpc-endpoint" }
}
```
or add an example like this, which is not correct, because `module.vpc.private_subnets` returns all subnets (ipv4 and ipv6), but not only ipv6
```
lakeformation = {
  service         = "lakeformation"
  ip_address_type = "ipv6"
  subnet_ids      = module.vpc.private_subnets
  tags            = { Name = "lakeformation-vpc-endpoint" }
},
```
so the question is - what do you think, is there any good workaround ?

Reply:
got it - we can forgo updating the example for this, its just adding one argument thats a direct pass through, not much to test/validate/demonstrate

Reply:
Removed the example